tests/integration/devfile/cmd_devfile_url_test.go

```diff
@@ -166,22 +171,6 @@ var _ = Describe("odo devfile url command tests", func() {
 			helper.MatchAllInOutput(stdout, []string{"URL " + url1 + " successfully deleted", "Applying URL changes"})
 		})
 
-		It("should create a url for a unsupported devfile component", func() {
```

Comment:
> Avoid shifting the code lines unnecessarily. I don't see any changes in this test case except for shifting of code lines.

The test has been shifted because it is specific to openshift but it was not skipped for kubernetes. So added the spec in the openshift specific context. Yes, we also follow such concept of not shifting test unnecessarily.